lib/iris/coords.py

```diff
@@ -1753,13 +1757,7 @@ def _sanitise_array(self, src, ndmin):
     @property
     def points(self):
         """Property containing the points values as a numpy array"""
-        if is_lazy_data(self._points):
-            self._points = as_concrete_data(self._points,
-                                            nans_replacement=np.ma.masked)
-            # NOTE: we probably don't have full support for masked aux-coords.
-            # We certainly *don't* handle a _FillValue attribute (and possibly
-            # the loader will throw one away ?)
-        return self._points.view()
+        return self._points_dm.data.view()
```

Comment:
I don't know. I read some comments from @pp-mo saying we shouldn't be returning views, but why not? Don't we want to prevent users altering the underlying data array? Even if the array is not writeable, they can still alter meta-data like shape and dtype.